regulations/static/regulations/js/source/views/drawer/toc-view.js

```diff
@@ -41,7 +41,7 @@ var TOCView = Backbone.View.extend({
     setActive: function(id) {
         var newActiveLink, subpart;
 
-        newActiveLink = this.$el.find('a[data-section-id="' + Helpers.findBaseSection(id) + '"]');
+        newActiveLink = this.$el.find('a[data-section-id="' + id + '"]');
```

Comment:
I don't understand the calling chain here, but where did this `id` get converted into a `tocId`?